### ISSUE TITLE
FIx kumo brocade switch

### ISSUE THREAD

**Issue by naved001** [CCI-MOC/ops-issues#517]
The switch can be pinged, but you can't SSH to it. It's at 10.0.21.24. 

May need an assist from someone at MGHPCC to power cycle it.

**Comment by naved001** [comment]
The host at `bu-23-16.maas.massopen.cloud` has a serial cable connected to this switch and we can power cycle the switch since we have PDU access.

To do the netinstall of brocade switches I follow this guide: https://extremeportal.force.com/ExtrArticleDetail?an=000085385

kzn-ipmi-gw has a tftp server and a nfs-server since those are required for firmware installation. It's tftp server worked fine, but the nfs-server wasn't working, running showmount on the kzn-ipmi-gw failed with this.
```
showmount clnt_create: RPC: Port mapper failure - Timed out
```

Anyway, I setup another nfs server on host 10.0.0.7 and with that I could finish installing the switch firmware.
The host 10.0.0.7 also backs up our switch configurations at /var/lib/tftpboo/. So I got the config for the failed switch from there and applied it.

The switch is up and running, however, a power9 node (bu-21-9) is still unreachable. So I will be spending some time looking at that.

**Comment by naved001** [comment]
Fixed this again for #552
